File: src/main/java/com/library/library/About.java
package com.library.library;

import javafx.fxml.FXML;
import javafx.scene.image.ImageView;

public class About {
    @FXML
    private ImageView img;

    public void showImage() {
        try {
            img.setImage(new javafx.scene.image.Image("/images/about.jpg"));
            img.setCache(true);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: src/main/java/com/library/library/About.kt
package com.library.library

import javafx.fxml.FXML
import javafx.scene.image.Image
import javafx.scene.image.ImageView

class About {
    @FXML
    private val img: ImageView? = null
    fun showImage() {
        try {
            img!!.image = Image("/images/about.jpg")
            img.isCache = true
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }
}